src/mount.ts

```diff
@@ -184,8 +193,10 @@ export function mount(
   if (global?.mocks) {
     const mixin = {
       beforeCreate() {
-        for (const [k, v] of Object.entries(global.mocks)) {
-          this[k] = v
+        for (const [k, v] of Object.entries(
+          global.mocks as { [key: string]: any }
+        )) {
+          ;(this as any)[k] = v
```

Comment:
TS is so ugly sometimes (or maybe due to Vue's dynamic nature it cannot be avoided...)